### DISCUSSION TITLE
Dutch translation

### DISCUSSION THREAD

**Discussion by HvF467** [aclap-dev/video-downloadhelper#1388] [Language & Translation]
[ Videao downloader vertaliig.txt](https://github.com/user-attachments/files/17477072/Videao.downloader.vertaliig.txt)

**Comment by Janou1** [comment]
Translated 45 missing strings

[nl(1).txt](https://github.com/user-attachments/files/17610607/nl.1.txt)

**Comment by Eegee** [comment]
To the submitters: please write "downloadgeschiedenis" as one word. Compound words should be written together in Dutch.

**Comment by Janou1** [comment]
and now with  "downloadgeschiedenis" as one word
[nl(1).txt](https://github.com/user-attachments/files/17648547/nl.1.txt)

**Comment by Rymnda** [comment]
So, I've now adjusted everything that needed to be changed, ensuring it's all in proper Dutch. Some things weren't described within the context of the Video DownloadHelper program, but rather generally. I hope this is useful for you all. I enjoyed working on it."

Here's a slightly more concise version:

I've made all the necessary adjustments to ensure the text is now in proper Dutch, taking into account the specific context of Video DownloadHelper. I hope this is helpful!

Currently, I only have Video DownloadHelper for Chrome and Opera, but not yet for Firefox. I was hoping it would work there, but unfortunately not. Are there any promotions for existing customers in this situation? Or perhaps a little something for my small contribution to the Dutch language corrections? 😉  ansemsr@gmail.com.  Cheers, Raymond

[nl.txt](https://github.com/user-attachments/files/17651367/nl.txt)

**Comment by HvF467** [comment]
**I agree a little something from video-downloadhelper for our small contributions to the Dutch language corrections is appreciated?**

**Comment by Basseken** [comment]
I translated a yet untranslated line:
[nl.txt](https://github.com/user-attachments/files/17782549/nl.txt)

Graag gedaan.

**Comment by QueenDennis** [comment]
[nl(1).txt](https://github.com/user-attachments/files/17791935/nl.1.txt)

Translated some of the still open lines for ya'll 

We benne op de wereld om elkaar te helpen nie waar

**Comment by rsschaap** [comment]
[nl.txt](https://github.com/user-attachments/files/18128484/nl.txt)
Nog wat vertaalde zinnetjes.

**Comment by spookrijder** [comment]
[nl.txt](https://github.com/user-attachments/files/18257350/nl.txt)

all missing strings translated by native dutch speaker, this should make the whole program available in Dutch.

**Reply by Eegee** [reply]
v9_panel_footer_show_history_tooltip = '''Laat download geschiedenis zien'''
download geschiedenis => downloadgeschiedenis

v9_history_page_title = '''Jouw download geschiedenis'''
download geschiedenis => downloadgeschiedenis

v9_history_no_recording_description = '''We slaan je downloadgeschiedenis niet op. Wil je dat Video Downloadhelper je download geschiedenis onthoud?'''
download geschiedenis => downloadgeschiedenis
onthoud => onthoudt

v9_settings_history_limit = '''Vergeet de download geschiedenis na X dagen.'''
download geschiedenis => downloadgeschiedenis
